src/seedu/addressbook/AddressBook.java

```diff
@@ -1144,14 +1144,17 @@ private static String getUsageInfoForExitCommand() {
      */
 
     /**
-     * Removes sign(p/, d/, etc) from parameter string
+     * Removes prefix from the given fullString if prefix occurs at the start of the string.
      *
-     * @param s  Parameter as a string
-     * @param sign  Parameter sign to be removed
-     * @return  string without the sign
+     * @param fullString  Parameter as a string
+     * @param prefix Parameter prefix to be removed
+     * @return  string without the prefix
      */
-    private static String removePrefixSign(String s, String sign) {
-        return s.replace(sign, "");
+    private static String removePrefix(String fullString, String prefix) {
```

Comment:
I believe you may have misunderstood the ideology behind refactoring. What you have done here is to improve the removePrefix method (by removing a prefix instead of a sign). You can refer to the list of examples in this link : https://refactoring.com/catalog/